test/curator.sh

```diff
@@ -133,26 +142,27 @@ update_config_and_restart() {
     # $2 - if present, expect errors
     if uses_config_maps ; then
         # use configmap
-        os::log::debug "$( oc delete configmap logging-curator )" || :
+        oc delete configmap logging-curator 2>&1 | artifact_out || :
         sleep 1
         if grep -q ^apiVersion: $1 ; then
-            os::log::debug "$( oc create -f $1 )" || : # oc get yaml dump, not a curator config file
+            oc create -f $1 2>&1 | artifact_out || : # oc get yaml dump, not a curator config file
         else
-            os::log::debug "$( oc create configmap logging-curator --from-file=config.yaml=$1 )"
+            oc create configmap logging-curator --from-file=config.yaml=$1 2>&1 | artifact_out
         fi
         sleep 1
     else
         # use secret volume mount
-        os::log::debug "$( oc delete secret curator-config${ops:-} )" || os::log::debug "no such secret curator-config${ops:-} - ignore"
-        os::log::debug "$( oc secrets new curator-config${ops:-} settings=$1 )"
-        os::log::debug "$( oc volumes dc/logging-curator${ops:-} --add --type=secret --secret-name=curator-config${ops:-} --mount-path=/etc/curator --name=curator-config --overwrite )"
+        oc delete secret curator-config${ops:-} 2>&1 | artifact_out || artifact_log no such secret curator-config${ops:-} - ignore
+        oc secrets new curator-config${ops:-} settings=$1 2>&1 | artifact_out
+        oc volumes dc/logging-curator${ops:-} --add --type=secret --secret-name=curator-config${ops:-} \
+            --mount-path=/etc/curator --name=curator-config --overwrite 2>&1 | artifact_out
     fi
     restart_curator ${2:-}
 }
 
 if uses_config_maps ; then
-    origconfig=`mktemp`
-    os::log::debug "$( oc get configmap logging-curator -o yaml > $origconfig )" || :
+    origconfig=$( mktemp )
+    oc get configmap logging-curator -o yaml > $origconfig 2>&1 | artifact_out || :
```

Comment:
`2>&1` would add output to stderr into $origconfig if any?  (or is it the purpose?)
I guess I'm worried too much... :)
```
oc get configmap logging-curator -o yaml > $origconfig 2| artifact_out || :
cat $origconfig | artifact_out || :
```